components/PostCardListItem/PostCardListItem.tsx

```diff
@@ -0,0 +1,215 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import GameSituation from "../../models/GameSituation";
+import getAgoByDate from "../../utilities/getAgoByDate";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import getSIMetricPrefixData from "../../utilities/getSIMetricPrefixData";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+
+interface Props extends React.Attributes {
+  post: Post;
+  isRecentPost?: boolean;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({
+  isRecentPost = false,
+  post,
+  ...props
+}: Props) {
+  const { heroIndex, playerCards } = post.gameSituation;
+  const { left, right } = playerCards[heroIndex]!;
+  const [nAgo, mobileTerm, term] = getAgoByDate(
+    isRecentPost ? post.createdAt : post.lastUpdatedAt
+  );
+
+  return (
+    <Root {...props}>
+      <PlayerHandArea>
+        <PlayerHandAreaBackGround />
+        <HandCard suit={left.suit} rank={left.rank} />
+        <HandCard suit={right.suit} rank={right.rank} />
+      </PlayerHandArea>
+      <PostTitle>
+        {post.title.length <= 65
+          ? post.title
+          : `${post.title.substring(0, 63)}...`}
+      </PostTitle>
+      <LikeArea>
+        <ThumbsUpIcon />
+        {post.likes}
+      </LikeArea>
+      <AttributeTitle>Play at</AttributeTitle>
+      <AttributeTitle>Ended at</AttributeTitle>
+      <AttributeTitle>Final Pod</AttributeTitle>
+      <AttributeTitle>{isRecentPost ? "Posted" : "Last Update"}</AttributeTitle>
+      <AttributeValue>
+        {getPositionByPlayerAndIndex(
+          post.gameSituation.playerLength,
+          post.gameSituation.heroIndex
+        )}
+      </AttributeValue>
+      <AttributeValue>
+        {post.gameSituation.river
+          ? Round.RIVER
+          : post.gameSituation.turn
+          ? Round.TURN
+          : post.gameSituation.flop
+          ? Round.FLOP
+          : Round.PREFLOP}
+      </AttributeValue>
+      <AttributeValue>{`${getSIMetricPrefixData(
+        getFinalPodOfTheGame(post.gameSituation)
+      )} BB`}</AttributeValue>
+      <AttributeValue>
+        {nAgo}
+        <MobileTermSpan>{mobileTerm}</MobileTermSpan>
+        <TermSpan>{term}</TermSpan>
+        &nbsp;ago
+      </AttributeValue>
+    </Root>
+  );
+}
+
+const Root = styled(Card)`
+  display: grid;
+  max-width: 440px;
+  grid-template-rows: 1fr 0.25fr 0.25fr;
+  grid-template-columns: 1.3fr 0.7fr 0.9fr 0.9fr 1.2fr;
+  row-gap: 4px;
+  column-gap: 8px;
+`;
+
+const PlayerHandArea = styled.div`
+  grid-row: 1 / 2;
+  grid-column: 1 / 2;
```

Comment:
Please define `grid-template-areas` at the parent and use `grid-area` at children. `grid-row` and `grid-column` have a huge problem at maintainability.